perk-store-services/src/main/java/org/exoplatform/perkstore/service/utils/NotificationUtils.java

```diff
@@ -335,7 +335,7 @@ public static final void setNotificationRecipients(NotificationInfo notification
       ignoredUsers.add(order.getSender().getId());
 
       // Retain in recipient list only users who are member of both ACL
-      addIdentityMembersFromProfiles(product.getMarchands(), recipientList);
+      addIdentityMembersFromProfiles(Collections.singleton(product.getReceiverMarchand()), recipientList);
```

Comment:
This will cause a regression, when we designated new managers for a product, they should be the ones who receive notifications.